app/src/test/java/com/ikarabulut/BoardTest.java

```diff
@@ -69,4 +72,137 @@ void setRandomGrid() {
         assertFalse(isDeadBoard);
     }
 
-}
\ No newline at end of file
+    @Test
+    @DisplayName("The board can bring a cell to life at a given location")
+    void setAliveAt_When1_1IsPassed_ReturnsTheCellObjectAt1_1() {
+        Board board = new Board();
+
+        Boolean cellStatus = board.isCellAliveAt(1, 1);
+        assertFalse(cellStatus);
+
+        board.setAliveAt(1, 1);
```

Comment:
I like that we send messages to Board to take care of which cells are alive at a given location. Board's job seems to be "Manage a collection of Cells", rather than "I'm a bag of Cells, here you go. You take them."

This prevents Law of Demeter violations later, and makes Board easier to collaborate with for other objects.